spi/max7219_32x8_spi/README.adoc

```diff
@@ -0,0 +1,40 @@
+= Attaching a Max7219 driving an 32x8 LED display via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to a Max7219 driving an 32x8 display. The particular device used is interfaced via SPI at 3.3v. This PCB uses 4 Max7219 devices, each driving an 8x8 LED matrix, giving the total of 32x8. These devices are designed to be cascaded together in this way, and SPI commands are passed down the line of devices as required.
+
+The Max7219 datasheet can be found here https://datasheets.maximintegrated.com/en/ds/MAX7219-MAX7221.pdf
+
+== Wiring information
+
+Wiring up the device requires 6 jumpers as follows:
+
+   * GPIO 16 (pin 21) MISO/spi0_rx-> DIN on max7219 board
```

Comment:
Probably worth using consistent capitalisation for Max7219 / max7219 ?

Reply:
And should this be spi0_tx -> DIN ? (which is what your `max7219_32x8_spi.c` says)

Reply:
Fixed.